lib/appliance_console.rb

```diff
@@ -505,6 +506,7 @@ module ApplianceConsole
           if database_configuration.activate
             database_configuration.post_activation
             say("\nConfiguration activated successfully.\n")
+            dbhost, dbtype, database = ApplianceConsole::Utilities.db_host_type_database
```

Comment:
It is in the outer most loop.
That code is always run since it is for showing the summary screen.

This is not necessary. Currently, the values don't get updated until after you logoff.
I added this to fix that bug.

1. I can create in another commit
2. I can delete
3. I can just leave as is (and even if it is broken, no harm no foul)